packages/image_picker/image_picker_for_windows/README.md

```diff
@@ -0,0 +1,36 @@
+# image\_picker\_for\_windows
+
+A Windows implementation of [`image_picker`][1].
+
+### pickImage()
+The arguments `maxWidth`, `maxHeight` and `imageQuality` are not supported.
+
+### pickVideo()
+The argument `maxDuration` is not supported on Windows.
+
+## Usage
+
+### Import the package
+
+This package is not yet [endorsed](https://flutter.dev/docs/development/packages-and-plugins/developing-packages#endorsed-federated-plugin), which means you need to add 
+not only the `image_picker`, as well as the `image_picker_for_windows`.
+
+### Use the plugin
+
+You should be able to use `package:image_picker` _almost_ as normal.
```

Comment:
What's the difference here relative to normal `image_picker` code?

Reply:
I just wanted to say that it is not fully featured yet. I'll change comment to make it clearer.